assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -104,18 +107,34 @@ const SelectableTitle = styled.span.attrs({ tabIndex: 0 })`
   cursor: pointer;
 `;
 
+const TitleTableCellContainer = styled.div`
```

Comment:
I know this is kinda crummy to have a div in a table, but this is the best way to do this. https://css-tricks.com/using-divs-inside-tables/